analysis/shamanenhancement/src/modules/apl/AplCheck.tsx

```diff
@@ -0,0 +1,78 @@
+import SPELLS from 'common/SPELLS';
+import { suggestion } from 'parser/core/Analyzer';
+import aplCheck, { build } from 'parser/shared/metrics/apl';
+import annotateTimeline from 'parser/shared/metrics/apl/annotate';
+import {
+  and,
+  buffMissing,
+  buffPresent,
+  buffStacks,
+  debuffMissing,
+  debuffPresent,
+  hasTalent,
+} from 'parser/shared/metrics/apl/conditions';
+
+/**
+ * Based on https://www.icy-veins.com/wow/enhancement-shaman-pve-dps-guide
+ */
+export const apl = build([
+  {
+    spell: SPELLS.LAVA_LASH,
+    condition: and(hasTalent(SPELLS.HOT_HAND_TALENT), buffPresent(SPELLS.HOT_HAND_BUFF)),
+  },
+  {
+    spell: SPELLS.FLAME_SHOCK,
+    condition: and(debuffMissing(SPELLS.FLAME_SHOCK)),
+  },
+  {
+    spell: SPELLS.FROST_SHOCK,
+    condition: and(
+      hasTalent(SPELLS.HAILSTORM_TALENT),
+      buffStacks(SPELLS.HAILSTORM_TALENT, { atLeast: 1 }),
+    ),
+  },
+  {
+    spell: SPELLS.STORMSTRIKE_CAST,
+    condition: and(hasTalent(SPELLS.STORMFLURRY_TALENT), buffPresent(SPELLS.STORMBRINGER_BUFF)),
+  },
+  {
+    spell: SPELLS.ELEMENTAL_BLAST_TALENT,
+    condition: and(buffStacks(SPELLS.MAELSTROM_WEAPON_BUFF, { atLeast: 5 })),
+  },
+  {
+    spell: SPELLS.LIGHTNING_BOLT,
+    condition: buffStacks(SPELLS.MAELSTROM_WEAPON_BUFF, { atLeast: 10 }),
+  },
+  {
+    spell: SPELLS.STORMSTRIKE_CAST,
+    condition: and(hasTalent(SPELLS.HOT_HAND_TALENT), buffMissing(SPELLS.HOT_HAND_BUFF)),
+  },
+  SPELLS.LAVA_LASH,
+  {
+    spell: SPELLS.SUNDERING_TALENT,
+    condition: and(hasTalent(SPELLS.SUNDERING_TALENT)),
+  },
```

Comment:
spells that aren't known are automatically hidden, so this can be simplified to:

```suggestion
  SPELLS.SUNDERING_TALENT,
```